packages/fiber/fiber.3.7.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Dune's monadic structured concurrency library"
+maintainer: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+license: "MIT"
+homepage: "https://github.com/ocaml-dune/fiber"
+bug-reports: "https://github.com/ocaml-dune/fiber/issues"
+depends: [
+  "dune" {>= "3.6"}
+  "ocaml" {>= "4.08"}
+  "ppx_expect" {with-test}
+  "dyn"
+  "stdune"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
    "@runtest" {with-test & ocaml:version >= "4.13"}
```
due to
```
#=== ERROR while compiling fiber.3.7.0 ========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/ocaml-dune/fiber/releases/download/3.7.0/fiber-lwt-3.7.0.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/fiber.3.7.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p fiber -j 255 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/fiber-7-c5ae92.env
# output-file          ~/.opam/log/fiber-7-c5ae92.out
### output ###
# File "fiber/test/fiber_tests.ml", line 1, characters 0-0:
# /usr/bin/git --no-pager diff --no-index --color=always -u _build/default/fiber/test/fiber_tests.ml _build/.sandbox/bc08e63157f9a7eac717e37ee33ab748/default/fiber/test/fiber_tests.ml.corrected
# diff --git a/_build/default/fiber/test/fiber_tests.ml b/_build/.sandbox/bc08e63157f9a7eac717e37ee33ab748/default/fiber/test/fiber_tests.ml.corrected
# index a55ee95..c162b4f 100644
# --- a/_build/default/fiber/test/fiber_tests.ml
# +++ b/_build/.sandbox/bc08e63157f9a7eac717e37ee33ab748/default/fiber/test/fiber_tests.ml.corrected
# @@ -66,7 +66,7 @@ let%expect_test "basics" =
#  
#  let%expect_test "collect_errors" =
#    test (backtrace_result unit) (Fiber.collect_errors (fun () -> raise Exit));
# -  [%expect {| Error [ { exn = "Stdlib.Exit"; backtrace = "" } ] |}]
# +  [%expect {| Error [ { exn = "Exit"; backtrace = "" } ] |}]
#  
#  let%expect_test "reraise_all" =
#    let exns =
# @@ -78,9 +78,9 @@ let%expect_test "reraise_all" =
#    [%expect
#      {|
#      Error
# -      [ { exn = "Stdlib.Exit"; backtrace = "" }
# -      ; { exn = "Stdlib.Exit"; backtrace = "" }
# -      ; { exn = "Stdlib.Exit"; backtrace = "" }
# +      [ { exn = "Exit"; backtrace = "" }
# +      ; { exn = "Exit"; backtrace = "" }
# +      ; { exn = "Exit"; backtrace = "" }
#        ] |}];
#    test (backtrace_result unit)
#      (Fiber.collect_errors (fun () ->
# @@ -91,9 +91,9 @@ let%expect_test "reraise_all" =
#      {|
#      finally
#      Error
# -      [ { exn = "Stdlib.Exit"; backtrace = "" }
# -      ; { exn = "Stdlib.Exit"; backtrace = "" }
# -      ; { exn = "Stdlib.Exit"; backtrace = "" }
# +      [ { exn = "Exit"; backtrace = "" }
# +      ; { exn = "Exit"; backtrace = "" }
# +      ; { exn = "Exit"; backtrace = "" }
#        ] |}];
#  
#    test unit ~expect_never:true
# @@ -166,7 +166,7 @@ let%expect_test "fill returns a fiber that executes before waiters are awoken" =
#  let%expect_test "collect_errors catches one error" =
#    test (backtrace_result unit) (Fiber.collect_errors failing_fiber);
#    [%expect {|
# -Error [ { exn = "Stdlib.Exit"; backtrace = "" } ]
# +Error [ { exn = "Exit"; backtrace = "" } ]
#  |}]
#  
#  let%expect_test "collect_errors doesn't terminate on [never]" =
# @@ -181,7 +181,7 @@ let%expect_test "failing_fiber doesn't terminate" =
#           let* () = failing_fiber () in
#           failing_fiber ()));
#    [%expect {|
# -Error [ { exn = "Stdlib.Exit"; backtrace = "" } ]
# +Error [ { exn = "Exit"; backtrace = "" } ]
#  |}]
#  
#  let%expect_test "collect_errors fail one concurrent child fibers raises" =
# @@ -190,7 +190,7 @@ let%expect_test "collect_errors fail one concurrent child fibers raises" =
#      (Fiber.collect_errors (fun () ->
#           Fiber.fork_and_join failing_fiber long_running_fiber));
#    [%expect {|
# -Error [ { exn = "Stdlib.Exit"; backtrace = "" } ]
# +Error [ { exn = "Exit"; backtrace = "" } ]
#  |}]
#  
#  let%expect_test "collect_errors can run concurrently" =
# @@ -200,7 +200,7 @@ let%expect_test "collect_errors can run concurrently" =
#         (fun () -> Fiber.collect_errors failing_fiber)
#         long_running_fiber);
#    [%expect {|
# -(Error [ { exn = "Stdlib.Exit"; backtrace = "" } ], ())
# +(Error [ { exn = "Exit"; backtrace = "" } ], ())
#  |}]
#  
#  let map_reduce_errors_unit ~on_error t =
# @@ -263,9 +263,9 @@ let%expect_test "handlers bubble up errors to parent handlers" =
#                     failing_fiber))));
#    [%expect
#      {|
# -    outer: raised Stdlib.Exit
# -    inner: raised Stdlib.Exit
# -    outer: raised Stdlib.Exit
# +    outer: raised Exit
# +    inner: raised Exit
# +    outer: raised Exit
#      Error () |}]
#  
#  let%expect_test "nested with_error_handler" =
# @@ -379,7 +379,7 @@ let%expect_test "sequential_iter error handling" =
#    test (unit_result unit) fiber ~expect_never:false;
#    [%expect {|
#      count: 1
# -    exn: Stdlib.Exit
# +    exn: Exit
#      finally
#      Error () |}]
#  
# @@ -409,7 +409,7 @@ let%expect_test _ =
#               Fiber.return (setter ())));
#    [%expect
#      {|
# -    Error [ { exn = "Stdlib.Exit"; backtrace = "" } ]
# +    Error [ { exn = "Exit"; backtrace = "" } ]
#      [PASS] Never raised as expected
#      [PASS] flag set |}]
#  
# @@ -429,8 +429,8 @@ let%expect_test _ =
#    [%expect
#      {|
#      Error
# -      [ { exn = "Failure(\"2\")"; backtrace = "" }
# -      ; { exn = "Failure(\"4\")"; backtrace = "" }
# +      [ { exn = "(Failure 2)"; backtrace = "" }
# +      ; { exn = "(Failure 4)"; backtrace = "" }
#        ]
#      [PASS] Never raised as expected
#      [PASS] flag set |}]
# @@ -675,7 +675,7 @@ let%expect_test "Stream.parallel_iter doesn't leak" =
#    (* Check that the number of live words keeps on increasing because we are
#       leaking memory: *)
#    test ~pred:( < ) ~iter_function:naive_stream_parallel_iter;
# -  [%expect {| [ 52; 115; 745; 7045 ] |}];
# +  [%expect {| [ 48; 111; 741; 7041 ] |}];
#    test ~pred:( = ) ~iter_function:Fiber.Stream.In.parallel_iter;
#    [%expect {| [ 43; 43; 43; 43 ] |}]
#  
# @@ -698,8 +698,8 @@ let%expect_test "fork - exceptions always thrown" =
#    [%expect
#      {|
#      Error
# -      [ { exn = "Failure(\"left\")"; backtrace = "" }
# -      ; { exn = "Failure(\"right\")"; backtrace = "" }
# +      [ { exn = "(Failure left)"; backtrace = "" }
# +      ; { exn = "(Failure right)"; backtrace = "" }
#        ] |}]
#  
#  let test iter =
# @@ -713,15 +713,15 @@ let%expect_test "parallel_iter - all exceptions raised" =
#    [%expect
#      {|
#      Error
# -      [ { exn = "Failure(\"1\")"; backtrace = "" }
# -      ; { exn = "Failure(\"2\")"; backtrace = "" }
# -      ; { exn = "Failure(\"3\")"; backtrace = "" }
# +      [ { exn = "(Failure 1)"; backtrace = "" }
# +      ; { exn = "(Failure 2)"; backtrace = "" }
# +      ; { exn = "(Failure 3)"; backtrace = "" }
#        ] |}]
#  
#  let%expect_test "sequential_iter - stop after first exception" =
#    test Fiber.sequential_iter;
#    [%expect {|
# -    Error [ { exn = "Failure(\"1\")"; backtrace = "" } ] |}]
# +    Error [ { exn = "(Failure 1)"; backtrace = "" } ] |}]
#  
#  let%expect_test "Stream: multiple readers is an error" =
#    (* [stream] is so that the first element takes longer to be produced. An
```